src/microbe_stage/Microbe.cs

```diff
@@ -725,6 +725,8 @@ public void ProcessSync(float delta)
 
         HandleReproduction(delta);
 
+        HandleInvulnerabilityDecay(delta);
```

Comment:
This should be put in the process async part as this doesn't require processing by the main thread.

Reply:
I moved this line to ProcessEarlyAsync... I think this is the correct place.